client/src/main/java/io/pravega/client/stream/StreamCut.java

```diff
@@ -67,4 +84,15 @@ static StreamCut fromBytes(ByteBuffer cut) {
         }
         return StreamCutInternal.fromBytes(cut);
     }
+
+    /**
+     * Deserializes a Base64 representation of the StreamCut obtained via {@link StreamCut#asText()}.
+     * @param base64String Base64 representation of StreamCut obtained using {@link StreamCut#asText()}
+     * @return The StreamCut object
+     * @throws SerializationException Exception during deserialization.
+     */
+    static StreamCut of(String base64String) throws SerializationException {
```

Comment:
please consider renaming this method to `from`

Reply:
corrected it.